editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -449,6 +445,26 @@ void Sprite2DEditor::_add_as_sibling_or_child(Node *p_own_node, Node *p_new_node
 	p_new_node->set_owner(get_tree()->get_edited_scene_root());
 }
 
+void Sprite2DEditor::_sync_sprite_resize_mode() {
+	if (node != nullptr) {
+		node->_editor_set_dragging_to_resize_rect(resize_region_rect->is_pressed());
+	}
+}
+
+void Sprite2DEditor::_update_sprite_resize_mode_button() {
+	if (node == nullptr) {
+		return;
+	}
+	resize_region_rect->set_disabled(!node->is_region_enabled());
+	resize_region_rect->set_pressed(node->_editor_is_dragging_to_resiz_rect());
+}
+
+void Sprite2DEditor::_update_sprite_resize_mode_tooltip() {
```

Comment:
I don't know how, but the tooltip and disabled state can still get desynced.
![image](https://github.com/user-attachments/assets/f2f71757-8c20-4ad0-94ed-eaa314666b0b)
Though as I mentioned, you can update them together. I don't see why there are 2 separate methods if they are called together and button's tooltip and disabled state basically depend on each other.

Reply:
It's still a separate method XD
Remove `_update_sprite_resize_mode_tooltip()` completely, it's not needed. You can set the tooltip in `_update_sprite_resize_mode_button()`.